lib/prima_auth0_ex/config_helper.ex

```diff
@@ -0,0 +1,10 @@
+defmodule PrimaAuth0Ex.ConfigHelper do
+  @moduledoc nil
+
+  def fetch_env_with_default(app, key, default) do
```

Comment:
scusa stavi cercando questa https://hexdocs.pm/elixir/Application.html#get_env/3 o non ho capito io? -.-"

Reply:
this

Reply:
oh grazie! ero abbastanza tiltato che non ci fosse nulla